### ISSUE TITLE
Documentation UI is not responsive

### ISSUE THREAD

**Issue by avigail-oron** [CloudSlang/CloudSlang.github.io#98]
Instead, when zooming in, a horizontal scroll appears which makes it very hard to read the documentation.
e.g. - http://www.cloudslang.io/#/docs#yaml-overview

**Comment by oritstone** [comment]
@avigail-oron is this still happening?

**Comment by avigail-oron** [comment]
Let me rephrase - it's not that it's not responding, The problem is that it's not a responsive one (that adjusts to the screen size)

**Comment by oritstone** [comment]
Fixed now that we moved our documentation to read-the-docs
